lib/IMAP/ImapMessageFetcher.php

```diff
@@ -520,6 +522,10 @@ private function parseHeaders(Horde_Imap_Client_Data_Fetch $fetch): void {
 					$this->unsubscribeUrl = $header->url;
 					break;
```

Comment:
uncomment these two lines to test mailing listst that have a http and mailto list